templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -27,8 +27,9 @@
 									<span class="octicon-check {{if not .Checked}}invisible{{end}}">{{svg "octicon-check"}}</span>
 									<span class="text">
 										{{avatar $.Context .User 28 "gt-mr-3"}}
-										{{.User.GetDisplayName}}
+										{{.User.Name}}
 									</span>
+									{{if $.IsShowFullName}}<span class="text" style="color: var(--color-text-light-2);">{{.User.FullName}}</span>{{end}}
```

Comment:
Could you change the inline styles to use CSS instead?

Reply:
Add a commit to put the inline css into less